templates/koji-hub/hub.conf.j2

```diff
@@ -89,9 +89,15 @@ Plugins = {% for plugin in koji_hub_plugins_list + koji_hub_noconfig_plugins_lis
 # KojiTraceback = normal
 
 ## These options are intended for planned outages
+{% if koji_hub_offline %}
+ServerOffline = True
+OfflineMessage = {{ koji_hub_outage_message }}
```

Comment:
Sounds good but {{ koji_hub_outage_message }} should then have at least a default value, and so just needs to be added to defaults/main.yml ? (just after the koji_hub_offline: boolean as a suggestion)

Reply:
@mohanboddu : let me know when you have it in this PR but ideally we open PR against `staging` branch so that we can then test in dev/staging env and when tested, merge back into main branch (used for prod automatically)

Reply:
@arrfab Fixed both of the changes you requested, please take a look.